features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/Comments/Comment.jsx

```diff
@@ -327,11 +335,56 @@ class Comment extends React.Component {
                                             allComments={allComments}
                                         />
                                     )} */}
+                                        {comment.replies && comment.replies.list.slice(0).map((reply, index) => (
```

Comment:
`slice(0)` is coping the entier array into a new array because the index given is 0 , Why do we need to slice the array here ?